packages/wdio-shadow-dom-service/src/index.js

```diff
@@ -0,0 +1,29 @@
+export default class ShadowDomService {
```

Comment:
I'm not an export with the shadow Dom, but shouldn't we also be able to pierce the shadow dom directly from the browser object?

Reply:
I'm not quite following. Do you mean that the browser object should have the same commands? document or body shouldn't have a shadowRoot property. There definitely could be a command that starts at either (I don't know where `$` starts it's queries) but it would just be an alias for `$`.